code/stable/dblpendulum/src/python/README.md

```diff
@@ -1,4 +1,6 @@
 # DblPendulum 
+> The program documented here is called DblPendulum.
```

Comment:
This text seems strange in a `README` file.  The `README` file doesn't document DblPendulum.  I feel like the `README` file would just start with the program name, possibly as a title (like #DblPendulum)

Reply:
You already have #DblPendulum.  Do we need the sentence at all?  If we need a sentence, I guess it might say something like "This program is named DblPendulum".  (Seems redundant though.)